src/ray/object_manager/object_manager.cc

```diff
@@ -225,6 +238,14 @@ void ObjectManager::TryPull(const ObjectID &object_id) {
   }
 };
 
+void ObjectManager::HandleObjectRequestCancelAbort(
+    const std::vector<ray::ClientID> &clients_to_request,
+    const std::vector<ray::ClientID> &clients_to_cancel, bool abort_object) {
+  // TODO(rkn): Implement this method. This method is responsible for issuing
```

Comment:
Is the plan to implement this in a follow-up PR? Although it might make this PR more complex, I think we should probably do it in this one because otherwise it's very hard to test the `PullManager`.